.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -274,6 +274,9 @@ onboarding_t0:
   - platform_tests/test_sequential_restart.py
   - platform_tests/test_xcvr_info_in_db.py
 
+onboarding_t0-2vlans:
+  - vlan/test_host_vlan.py
+  - vlan/test_vlan_ping.py
```

Comment:
We do not have many scripts to be covered in t0-2vlans scenario. Probably not necessary to add a new optional checker for t0-2vlans. Can you just add these 2 scripts to the existing t0-2vlans checker? The new optional job is not required IMO.

Reply:
For these 2 tests, it maybe fine, but we still have a pending script test_vlan.py, it's continuously failing on t0-2vlans, so once we fixed error and add it to PR test, we need to observe some time before adding to t0-2vlans checker.
So I think if adding a new checker is needed in the future, why not add it now?

Reply:
Fixed